src/models/slices/vertical.ts

```diff
@@ -1,11 +1,34 @@
-import { VerticalSearchResponse, AutocompleteResponse, VerticalResults } from '@yext/answers-core';
+import {
+  AppliedQueryFilter,
+  AutocompleteResponse,
+  DisplayableFacet,
+  Result,
+  SearchIntent,
+  SortBy,
+  Source,
+  VerticalResults
+} from '@yext/answers-core';
+
+export interface AllResultsForVertical {
+  facets: DisplayableFacet[],
+  results: Result[],
+  resultsCount: number,
+  searchIntents: SearchIntent[]
+}
+
 export interface VerticalSearchState {
-  alternativeVerticals?: VerticalResults[]
-  key?: string,
+  allResultsForVertical?: AllResultsForVertical,
+  alternativeVerticals?: VerticalResults[],
+  appliedQueryFilters?: AppliedQueryFilter[],
   autoComplete?: AutocompleteResponse,
-  results?: VerticalSearchResponse,
   displayName?: string
   limit?: number,
   offset?: number,
-  searchLoading?: boolean
+  queryDurationMillis?: number,
+  results?: Result[],
+  resultsCount?: number,
```

Comment:
Minor nit: can we get rid of this? If they have the array of Results, they can deduce the count.

Reply:
I believe we need to have it separately because there could be a resultsCount of 500 but the backend only returns 20 of them

Reply:
Ahh, that's right! Nvmd